.gitignore

```diff
@@ -1,7 +1,8 @@
 inviwo-log.html
 **/.DS_Store
-tools/doxygen/doc
-tools/doxygen/doc-qt
 *.pyc
 pyconfig.ini
-.idea
\ No newline at end of file
+.idea
+*.qch
+build
```

Comment:
Don't put build in here, one should nerver build in source, we should not encourage that